types/boom/index.d.ts

```diff
@@ -32,14 +34,17 @@ declare namespace Boom {
         /** headers - an object containing any HTTP headers where each key is a header name and value is the header content. (Limited value type to string https://github.com/hapijs/boom/issues/151 ) */
         headers: {[index: string]: string};
         /** payload - the formatted object used as the response payload (stringified). Can be directly manipulated but any changes will be lost if reformat() is called. Any content allowed and by default includes the following content: */
-        payload: {
-            /** statusCode - the HTTP status code, derived from error.output.statusCode. */
-            statusCode: number;
-            /** error - the HTTP status message (e.g. 'Bad Request', 'Internal Server Error') derived from statusCode. */
-            error: string;
-            /** message - the error message derived from error.message. */
-            message: string;
-        }
+        payload: IPayload;
+    }
+    export interface IPayload {
```

Comment:
Now I'm getting the previous error I got of:

    > definitely-typed@0.0.1 lint /Users/ajp/tmp/DefinitelyTyped
    > dtslint types "boom"

    Object.entries is not a function

    npm ERR! Darwin 15.6.0
    npm ERR! argv "/usr/local/Cellar/node@6/6.10.1/bin/node" "/usr/local/bin/npm" "run" "lint" "boom"
    npm ERR! node v6.10.1
    npm ERR! npm  v4.1.2
    npm ERR! code ELIFECYCLE
    npm ERR! definitely-typed@0.0.1 lint: `dtslint types "boom"`
    npm ERR! Exit status 1
    npm ERR! 
    npm ERR! Failed at the definitely-typed@0.0.1 lint script 'dtslint types "boom"'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the definitely-typed package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     dtslint types "boom"
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs definitely-typed
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls definitely-typed
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     /Users/ajp/tmp/DefinitelyTyped/npm-debug.log